en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
+gui.xaero_world_teleport_command=Teleport Command
 gui.xaero_use_default=Use Default
-gui.xaero_waypoint_name=Waypoint Name
-gui.xaero_centered_enlarged=Center When Enlarged
-gui.xaero_zoomed_out_enlarged=Zoom Out When Enlarged
+gui.xaero_centered_enlarged=Center Minimap When Enlarged
+gui.xaero_zoomed_out_enlarged=Zoom Out Minimap When Enlarged
 gui.xaero_sort=Sort By
 gui.xaero_sort_reversed=Reversed Sort Order
 gui.xaero_sort_unsorted=Unsorted
 gui.xaero_sort_name=Name
-gui.xaero_sort_symbol=Initial
+gui.xaero_sort_symbol=Initials
 gui.xaero_sort_distance=Distance
-gui.xaero_box_cave_maps=Mapping mode that can go deeper than the surface blocks, mainly to display underground caves and interiors of buildings. The roof size stands for the size of a solid horizontal "square" of blocks that needs to be detected above you to activate the cave mode.
+gui.xaero_box_cave_maps=Mapping mode that can go deeper than the surface blocks, mainly to display underground caves and interiors of buildings. The ceiling size refers to the size of a horizontal "square" of solid blocks that has to be detected above you to activate the cave mode.
 gui.xaero_box_entity_amount=The limit per each entity category.
-gui.xaero_box_distance=Which ingame waypoints to show the distance to for.
-gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This does not override the "Distance To WP" option.
-gui.xaero_box_arrow_scale=Scale for the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_lock_player_heads=Display player heads instead of coloured dots even if the TAB key is not pressed.
-gui.xaero_box_cave_zoom=How much to zoom further in when the cave maps mode is active.
-gui.xaero_box_entity_depth=Display entities darker depending on their Y level relative to you.
-gui.xaero_box_height_limit=Maximum Y level distance for entities until they are no longer displayed.
-gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.,
-gui.xaero_toggle_map_waypoints=Toggle On-map Waypoints
-gui.xaero_entity_nametags=Entity Name Tags
+gui.xaero_box_distance=Specifies, for which in-game waypoints to show the distance to.
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=How much to zoom further in the minimap when the cave mode is active.
```

Comment:
I'm not entirely sure that "the minimap" is in the correct position in this sentence. If you know, could you help me, please?

Reply:
Do you think this is better?
How much to zoom the minimap further in when the cave mode is active.

Reply:
Your version definitely sounds more fluent than mine, so because I couldn't find any reliable grammar rule about such relatively complex phrase, I'm going to use yours.